assets/javascripts/discourse/components/discourse-post-event/invitees.gjs

```diff
@@ -0,0 +1,70 @@
+import Component from "@glimmer/component";
+import { fn } from "@ember/helper";
+import { action } from "@ember/object";
+import { service } from "@ember/service";
+import DButton from "discourse/components/d-button";
+import { popupAjaxError } from "discourse/lib/ajax-error";
+import i18n from "discourse-common/helpers/i18n";
+import PostEventInvitees from "../modal/post-event-invitees";
+import Invitee from "./invitee";
+
+export default class DiscoursePostEventInvitees extends Component {
+  @service discoursePostEventApi;
+  @service store;
+  @service modal;
+
+  @action
+  async showAllInvitees(event) {
```

Comment:
since this is called only with `@event` I'd remove the arg and used `this.args.event` in the function directly